neo/app/controllers/Application.scala

```diff
@@ -59,9 +59,11 @@ class Application @Inject()(val wsClient: WSClient,
     ActorFlow.actorRef(out => UserActor.props(out, cachesActor, USGeoGnosis))
   }
 
-  def tweet(id:String) = Action {
+  def tweet(id:String) = Action.async {
```

Comment:
It's actually a pretty good practice to learn the code through code review. @JeremyLi28 @xixZ  you are very encouraged to ask questions about the unknown codes 😄